NetworkRecurrentLayer.py

```diff
@@ -130,6 +130,142 @@ def step(self, i_t, x_t, z_t, y_p, c_p, *other_args):
     # return: next outputs (# unit.n_act, y_t, c_t, ...)
     return (y_t * i_output, c_t * i_h + c_p * (1 - i_h)) + tuple(other_outputs)
 
+class Leaky(Unit):
+  """
+  A 1D cell proposed in http://jmlr.org/papers/volume17/14-203/14-203.pdf
+  The simplified equations can be seen in Table 7, page 36.
+  Type A with gamma_3==0.
+  This cell has 3 units instead of 4 like LSTM
+  """
+  def __init__(self, n_units, **kwargs):
+    super(LSTME, self).__init__(
```

Comment:
Please provide the correct class name in the call of the super constructor (LSTME -> LEAKY)